tools/downloader/common.py

```diff
@@ -537,6 +538,8 @@ def deserialize(cls, model, name, subdirectory):
 
                 precisions = set(files_per_precision.keys())
 
+            quantized = model.get('quantized', None)
```

Comment:
Please validate the field: it should be either `"INT8"` or `None`. Also, check that if this is specified, then `model_optimizer_args` is also specified.

Reply:
validation added (also move getting values a bit above)